examples/mitqa/README.md

```diff
@@ -0,0 +1,44 @@
+
+#### Replicating leaderboard results for hybridqa
+- Download trained `row retriever` model from [here](https://huggingface.co/PrimeQA/MITQA_hybridqa_row_retriever/resolve/main/row_retriever.bin) and copy it into the `data/hybridqa/models/row_retriever/`.
+- Check the config file to make sure the `model_name_path_ae` hyperparameter is set correctly to `PrimeQA/MITQA_hybridqa_multi_answer_answer_extractor`
+- Run `python primeqa/mitqa/run_mitqa.py <inference_config_file_path>`
+
+### Training on OTTQA dataset
+
+- Create data root folder named `data/ottqa`.
+- Get data from this (https://github.com/wenhuchen/OTT-QA/tree/master/released_data) repo and copy to the data root folder.
+- Create sub folders for row retriever and answer extractor models as `data/ottqa/models/table_retriever`, `data/ottqa/models/link_generator`, `data/ottqa/models/row_retriever` and `data/ottqa/models/answer_extractor`.
+- Get passages and tables corpuses using the following commands:
+`wget https://opendomainhybridqa.s3-us-west-2.amazonaws.com/all_plain_tables.json`
+`wget https://opendomainhybridqa.s3-us-west-2.amazonaws.com/all_passages.json`
+- Make sure tables and passages corpuses are in data root folder.
+- Make sure preprocessed triples for training and evaluation of the table retrievers are in data root folder.
+#### Replicating leaderboard results for OTTQA
+- Create two sub folders `qry_encoder` and `ctx_encoder` inside `data/ottqa/models/table_retriever`.
+- Download trained `query encoder` model and config file from [here](https://huggingface.co/PrimeQA/MITQA_OTTQA_DPR_Table_Retriever_Query_Encoder/tree/main) and copy it into `data/ottqa/models/table_retriever/qry_encoder`.
+- Download trained `context encoder` model and config file from [here](https://huggingface.co/PrimeQA/MITQA_OTTQA_DPR_Table_Retriever_Context_Encoder/tree/main) and copy it into `data/ottqa/models/table_retriever/ctx_encoder`.
+- Download trained `link predictor` model from [here](https://huggingface.co/PrimeQA/MITQA_OTTQA_Link_Predictor/resolve/main/model-ep9.pt) and copy it into the path `data/ottqa/models/link_generator`
+- Download trained `row retriever` model from [here](https://huggingface.co/PrimeQA/MITQA_hybridqa_row_retriever/resolve/main/row_retriever.bin) and copy it into `data/hybridqa/models/row_retriever/`.
+- Check the config file to make sure the `model_name_path_ae` hyperparameter is set correctly to `PrimeQA/MITQA_hybridqa_multi_answer_answer_extractor`
+- Run `python primeqa/mitqa/run_mitqa.py <ottqa_inference_config_file_path>`
+
+#### To train the table retriever model for OTTQA dataset
+- Make sure to set the flag `train_tr` in the config file `primeqa/mitqa/config/train_ottqa.json` and also check and update all other hyperparameter values if required.
```

Comment:
I don't see `train_tr` in the train_ottqa.json .  So maybe should state add the flag ...
What does "update all other hyperparameter values if required." mean ? what is the recommended values

Reply:
table retrieval training is optional as we are already providing the model. But if someone wants to train the table retriever they can set this flag to true.